e2e/.eslintrc

```diff
@@ -1,25 +0,0 @@
-{
-	"folders": [
-		{
-			"path": "."
-		}
-	],
-	"settings": {
-		"editor.codeActionsOnSave": {
-			"source.fixAll.eslint": true,
-			"source.fixAll.markdownlint": true
-		},
-		"[javascript]": {
-			"editor.tabSize": 2,
-			"editor.insertSpaces": true,
-			"editor.detectIndentation": false,
-			"editor.formatOnPaste": true,
-			"editor.formatOnSave": true
-		},
-		"markdownlint.config": {
-			"default": true,
-			"MD034": false,
-			"MD046": false
-		}
-	}
```

Comment:
why is this getting deleted?

Reply:
This should be the content of a `.code-workspace` file, not an ES Lint configuration. It now exists in the top level directory. For ES Lint config, we have `.eslintrc.js`.